app/views/ConfirmationPageView.scala.html

```diff
@@ -25,19 +26,34 @@
     h1: components.h1
 )
 
-@(email: Option[Email], fileRole: FileRole, returnLink: String)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+@(email: Option[Email], fileRole: FileRole, returnLink: String, helper: CheckYourAnswersHelper)(
```

Comment:
Better to have below 4 parameters on separate lines to increase the readability as its difficult to read as is

email: Option[Email], fileRole: FileRole, returnLink: String, helper: CheckYourAnswersHelper